posts/app.py

```diff
@@ -169,7 +169,7 @@ def generate_timestamp():
     Returns:
         datetime: the current time.
     """
-    return datetime.datetime.now()
+    return datetime.datetime.utcnow().isoformat(" ", "seconds")
```

Comment:
maybe a named parameter for  delimiter=" " or whatever this means

Reply:
Done https://github.com/knative-portability/large-events/pull/45/commits/a88de4b2953206978ee4c6a11b853ea9f29b65ed